grafana-plugin/src/containers/OutgoingWebhookForm/OutgoingWebhookForm.types.ts

```diff
@@ -15,4 +15,11 @@ export const WebhookFormFieldName = {
   ForwardAll: 'forward_all',
   Data: 'data',
 } as const;
+
 export type WebhookFormFieldName = (typeof WebhookFormFieldName)[keyof typeof WebhookFormFieldName];
+
+export interface TemplateParams {
+  name: WebhookFormFieldName;
+  value: any;
```

Comment:
Why is it `any`? It seems we always invoke it with the `string`, can we use `string` instead of `any` in this case?